chef/data_bags/crowbar/bc-template-glance.schema

```diff
@@ -76,6 +76,7 @@
             "rbd": {
               "type": "map", "required": true, "mapping": {
                 "store_ceph_conf": { "type": "str", "required": true },
+                "store_admin_keyring": { "type": "str", "required": true },
```

Comment:
@vuntz : Ok, where would you like to hard-code this values? Directly in recipe code or in `chef/cookbook/glance/attributes/default.rb`?